packages/build-info/src/node/file-system.ts

```diff
@@ -11,6 +11,10 @@ export class NodeFS extends FileSystem {
     this.cwd = process.cwd()
   }
 
+  getEnvironment() {
+    return 'node' as const
```

Comment:
We could use an enum instead of the string

```
enum Environment {
 BROWSER = 'browser',
 NODE = 'node',
}
```

Reply:
Implemented in https://github.com/netlify/build/pull/4876/commits/6561b05307bde3490e54d00197aa8f75a97a04c9